sub-logs/cat.json

```diff
@@ -0,0 +1,17 @@
+{
+    "repo": "",
+
+    "owner": {
+      "username": "xxCopyCat",
+      "email": "ibrahimbode47@gmail.com"
+    },
+
+    "target": {
+      "CNAME": {
+         "name": "CNAME", 
```

Comment:
This should be the same as the file name.

Reply:
Ah alright got it... Do I create a new pull request? I'm not sure how to manage with github.